apps/tup-ui/src/main.global.css

```diff
@@ -12,9 +12,14 @@ body > main {
 .o-site {
   display: flex;
   flex-direction: column;
-
+  overflow: auto;
   flex-grow: 1;
 }
 .o-site > * {
   flex-grow: 1;
 }
+
+#cms-content {
+  margin-bottom: 0px;
```

Comment:
I think this is not necessary, because https://github.com/TACC/tup-ui/blob/940250569a6e885ec0e52c3a8f4cec4e008e4f0a/apps/tup-cms/src/apps/dashboard/templates/dashboard/assets.html#L9-L12

```suggestion
```

Reply:
| new style (excess?) | existing style (same task) |
| - | - 
| ![new style to remove margin](https://user-images.githubusercontent.com/62723358/207162097-d236d057-b936-4760-8077-8783f2ed447a.png) | ![existing style to unset margin](https://user-images.githubusercontent.com/62723358/207162100-ca71e582-b661-4086-90d3-bcd3f7c5e6a2.png)